src/models/ResponseModel.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * HTTP headers to be included in all responses.
+ */
+export const RESPONSE_HEADERS = {
+  'Content-Type': 'application/json',
+  /** Required for CORS support to work */
+  'Access-Control-Allow-Origin': '*',
+  /** Required for cookies, authorization headers with HTTPS */
+  'Access-Control-Allow-Credentials': true,
+};
+
+/**
+ * Default message provided as part of response.
+ */
+export const DEFAULT_MESSAGE = 'success';
+
+/**
+ * Our standard response model for HTTP endpoints.
+ */
+export default class ResponseModel {
+  body: any;
+
+  code: any;
```

Comment:
Could we use `statusCode` instead, just for clarity?

I think I mentioned this one before, and again can't remember what was discussed.

Reply:
I agree, but am saving this for a future PR – trying to keep this one focussed on TypeScript and redesigning DI with as few other changes as possible!